controller.py

```diff
@@ -514,6 +519,12 @@ def get_argument_parser():
         action="store_true",
         required=False,
     )
+    parser.add_argument(
+        "--goldenrun-only",
+        help="Only run goldenrun to create the goldenrun backup file",
+        action="store_true",
+        required=False,
+    )
```

Comment:
More a question than a review comment:
Should we somehow disallow append and goldenrun-only to be both set by the user?

@lukasauer what do you think? necessary or obvious that it does not really make sense to have both flags set?

Reply:
It does not break anything having both active, but of course, it does not really make sense. From the output it should be clear though that nothing got simulated, so I think we do not have to disallow having both set.